ajax/libs/venn.js/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "venn.js",
+  "filename": "venn.min.js",
+  "description": "A javascript library for laying out area proportional venn and euler diagrams",
+  "homepage": "https://github.com/benfred/venn.js/",
+  "author": "Ben Frederickson",
```

Comment:
Let's make this field as the same as upstream's info.
`"author": "Ben Frederickson (ben@benfrederickson.com)"`

Ref: https://github.com/benfred/venn.js/blob/master/bower.json#L7
Thanks